internal/tracinginterceptor/interceptor.go

```diff
@@ -177,12 +177,60 @@ func (i *Interceptor) CallOneway(ctx context.Context, req *transport.Request, ou
 
 // HandleStream implements interceptor.StreamInbound
 func (i *Interceptor) HandleStream(s *transport.ServerStream, h transport.StreamHandler) error {
-	panic("implement me")
+	parentSpanCtx, _ := i.tracer.Extract(i.propagationFormat, getPropagationCarrier(s.Request().Meta.Headers.Items(), i.transport))
+	span := i.tracer.StartSpan(
+		s.Request().Meta.Procedure,
+		opentracing.ChildOf(parentSpanCtx),
+		ext.SpanKindRPCServer,
+		opentracing.StartTime(time.Now()),
+	)
+	defer span.Finish()
+
+	// Wrap the ServerStream in a tracedServerStream
+	tracedStream := newTracedServerStream(*s, span, i.log)
+	err := h.HandleStream(tracedStream.ServerStream)
```

Comment:
What's the point of tracedStream wrapper? If you delete it, it'd change nothing, right?